### PR TITLE
Fix recursive case for -(1.s)

### PR DESCRIPTION
**Pull request by MasseGuillaume**

### CODE REVIEW

File: tests/slow/coverage.txt [hunk 1]
```diff
@@ -10,5 +10,4 @@ scala/test/files/pos/t9050.scala
 scala/test/files/run/t2378.scala
 scala/test/files/specialized/arrays.scala
 scalaz/core/src/main/scala/scalaz/Foldable.scala
-shapeless/core/src/main/scala/shapeless/ops/zipper.scala
```
Comment (MasseGuillaume) [inline tests/slow/coverage.txt:13]: 13!

File: format/src/main/scala/org/scalafmt/internal/TokenOps.scala [hunk 2]
```diff
@@ -86,7 +87,7 @@ object TreeOps {
       true
 
     case (g.Term.PrefixExpr("-"), g.Term.PrefixArg(Term.Select(tree, _), _))
```
Comment (olafurpg) [inline format/src/main/scala/org/scalafmt/internal/TokenOps.scala:89]: Should we not deconstruct Term.Select here now that it's handled in startsWithNumericLiteral?

**Review by olafurpg — APPROVED** [review]
LGTM 👍 Nice!

File: format/src/main/scala/org/scalafmt/internal/TokenOps.scala (hunk 2 above)
Comment (MasseGuillaume) [reply format/src/main/scala/org/scalafmt/internal/TokenOps.scala:89]: I want it to fall in the other case if it's not a Select.

**Pull request merged**